client/a8c-for-agencies/sections/overview/body/intro-cards/index.tsx

```diff
@@ -1,6 +1,6 @@
 import { Card } from '@automattic/components';
+import DotPager from '@automattic/components/src/dot-pager';
```

Comment:
This should be importing from a true top-level export in `@automattic/components`. If that isn't working, we need to fix the export.
```suggestion
import { Card, DotPager } from '@automattic/components';
```

Reply:
It works, but I learned recently that the tree shaking sometimes fails if you import like that, and you end up pulling the whole package.

Reply:
It's intentional, but not 100% sure if good. 

Hi @sgomes! As I learned recently, thanks to you, it's better to import the file you want exactly rather than an umbrella `index` file, does this apply to packages in your opinion?

See: p1716996185662019/1716994970.139089-slack-C4GAQ900P

Reply:
Hey @alshakero! 👋

It's okay to use an umbrella `index` file, as long as:
- Exports are sensibly distributed across their own modules
- The index file is as clean as possible, ideally only re-exporting things
- The package has a `sideEffects` declaration. This is usually `sideEffects: false` (indicating nothing has side effects) or a glob of CSS/SCSS files, indicating only styles have side effects.

Some packages are better than others at following these rules, so as an importer I'd keep an eye out 🙂

Reply:
That's great. Thanks!